scraper/scraper.js

```diff
@@ -159,3 +159,10 @@ function addTermsToData(terms, data) {
     data[key].push(termData);
   }
 }
+
+function normalizeKey(key, type) {
+  if (type === "enum-value") {
```

Comment:
switch? in case we need to do more later?

Reply:
Maybe not yet. We can always rewrite when needed 😄 
otherwise, we'll have a switch with single case till then 😆

Reply:
We can probably then just inline this

Reply:
type === "enum-value" ? key.replace(/^"|"$/g, "") : key

Reply:
too late 😛